src/hooks/services/hooks.service.ts

```diff
@@ -276,6 +276,7 @@ export class HooksService {
           ),
           accessToken,
           this.aggregator,
+          AccountType.CHECKING,
```

Comment:
Why not using `acccount.type`? It has already mapped before in the `mapBridgeAccountV2`.

Or maybe by security:

```suggestion
          account.type ?? AccountType.CHECKING,
```